cli.py

```diff
@@ -1,10 +1,11 @@
 import ftplib
 import io
 import sys
-import pydantic
-import tum_esm_utils
+
 import click
+import pydantic
 import em27_metadata
+import tum_esm_utils
 
 _RETRIEVAL_ENTRYPOINT = tum_esm_utils.files.rel_to_abs_path("src", "retrieval", "main.py")
```

Comment:
Please do not modify the order of the imports if they don't change. In all the files. Reformatting without changing anything makes it harder to track, when a file/a section has been actually changed.

Reply:
I updated some of the files I worked on to follow the PEP 8 style guide for imports (https://peps.python.org/pep-0008/#imports), namely grouping imports by type and separating them by a space. I also alphabetically organised them within groups. Are you sure you want me to revert these updates?

Reply:
Yes, that makes sense. Thanks!